libgnucash/engine/Transaction.c

```diff
@@ -327,12 +326,10 @@ gnc_transaction_get_property(GObject* object,
         g_value_take_object(value, tx->common_currency);
         break;
     case PROP_POST_DATE:
-        ts.tv_sec = tx->date_posted;
-        g_value_set_boxed(value, &ts);
+        g_value_set_boxed(value, &tx->date_posted);
```

Comment:
It's a C file, so it would have to be ```(Time64*) ``` rather than ```static_cast<Time64*>```, but I think if it was C++ it wouldn't even let me do ```reinterpret_cast<Time64*>```. Probably better to just write it correctly instead of playing pointer games.